packages/lib-react-components/src/ZooFooter/ZooFooter.spec.jsx

```diff
@@ -4,7 +4,6 @@ import ZooFooter from './ZooFooter'
 
 
 describe('<ZooFooter />', function () {
-  this.timeout(5000)
```

Comment:
`this` does not exist in the Vitest env. It's timeout config is located in vitest.config.js.